tests/StackExchange.Exceptional.Tests/StackExchange.Exceptional.Tests.csproj

```diff
@@ -10,7 +10,9 @@
     <ProjectReference Include="..\..\src\StackExchange.Exceptional.MySQL\StackExchange.Exceptional.MySQL.csproj" />
     <ProjectReference Include="..\..\src\StackExchange.Exceptional.PostgreSql\StackExchange.Exceptional.PostgreSql.csproj" />
     <ProjectReference Include="..\..\src\StackExchange.Exceptional.MongoDB\StackExchange.Exceptional.MongoDB.csproj" />
-    <PackageReference Include="Jil" Version="2.17" />
+    <PackageReference Include="Jil" />
+    <PackageReference Include="xunit"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
```

Comment:
The last 2 here can be global - they're going to be present in all tests :)

Reply:
Yes, at first I tried to put it as GlobalPackageReference in Directory.Packages.props but this make the assembly added with `IncludeAssets="Runtime;Build;Native;contentFiles;Analyzers"` (and so not referenced as a compile time assembly)  so either :
- I add package reference in each csproj insted (which I chose)
- I add package reference with Update in each csproj
- I dont use GlobalPackageReference and declare it in Directory.Build.props (which I didn't pick because it makes three places to declare PackageReference)
Tell me which you want to go with, I'll update accordingly ;)

Reply:
Ahhh I gotcha - no worries if that's giving any trouble at all 👍to current :)